.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+.idea/
+Cargo.lock
```

Comment:
Why are you adding `Cargo.lock` to .gitignore? This should remain checked-in for program crates.

Reply:
It is in the `.gitignore` of the `clock-bound-c` and `clock-bound-d` folder before they are merged into one.

Reply:
Will remove it.